### DISCUSSION TITLE
Does mapperly support generating a DTO?

### DISCUSSION THREAD

**Discussion by moxplod** [riok/mapperly#457] [Q&A]
I tried looking around in the documentation and then I saw the comment here: https://amanagrawal.blog/2021/07/28/using-c-sharp-source-generators-to-generate-dtos-part2/

Does mapperly support generating a DTO from a given class (entity class or otherwise)?

**Comment by latonz** [comment]
This is not supported by Mapperly and is out of scope as Mapperly focuses on object to object mappings.
